xblock/test/test_fields.py

```diff
@@ -703,7 +703,7 @@ def test_setting_the_same_value_marks_field_as_dirty():
     # pylint: disable=unsubscriptable-object
     class FieldTester(XBlock):
         """Test block for set - get test."""
-        non_mutable = String(scope=Scope.settings)
+        # non_mutable = String(scope=Scope.settings)
```

Comment:
Why have you commented them rather than removing them?

Reply:
i mistakenly left them, i have removed them.